scripts/update_headers.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2020 Contributors to the Pyro project.
```

Comment:
@neerajprad here is the new script file

Reply:
Thanks, @fritzo. Should we do this for the other projects in pyro-ppl too?

Reply:
Yes, we should do this for all repos in https://github.com/pyro-ppl.

Reply:
Okay, I'll make the changes for numpyro and pyro-api.